relnote(Fx145): layout.css.anchor-positioning.enabled flipped true in Nightly (#41643)
mdn/content: files/en-us/mozilla/firefox/experimental_features/index.md

| Release channel   | Version added | Enabled by default? |
| ----------------- | ------------- | ------------------- |
| Nightly           | 145           | Yes                 |
| Developer Edition | 131           | No                  |
| Beta              | 131           | No                  |
| Release           | 131           | No                  |

- `layout.css.anchor-positioning.enabled`
  - : Set to `true` to enable.

#### `anchor-size()` function

> The CSS {{CSSXRef("anchor-size")}} function enables setting anchor-positioned element's size, position, and margins relative to the dimensions of anchor elements. ([Firefox bug 1972610](https://bugzil.la/1972610)).

Reviewer: ```suggestion
The CSS {{CSSXRef("anchor-size")}} function lets you set an anchor-positioned element's size, position, and margins relative to the dimensions of its anchor element. ([Firefox bug 1972610](https://bugzil.la/1972610)).
```
we can do a follow up to also update https://developer.mozilla.org/en-US/docs/Web/CSS/anchor-size.
